src/nbl/video/IOpenGLPipeline.h

```diff
@@ -73,7 +73,7 @@ class IOpenGLPipeline
 
 		        uint8_t* valueptr = state+m.offset;
 
-                uint32_t arrayStride = 0u;
+                uint32_t arrayStride = 0u; // TODO: @Crisspl this calculation is 100% wrong with `row_major` and arrays of matrices (for both std140 and std430)
```

Comment:
is it confirmed that it's spirv-cross bug?

Reply:
no idea, please investigate

Reply:
nvm, it's already fixed in introspector

Reply:
so why was it something like 32 or 16 for a mat4x3 in row major?

Reply:
do a test with Push Constants and arrays of matrices... just to make sure row-major and column-major work properly

Reply:
> so why was it something like 32 or 16 for a mat4x3 in row major?

because i was trating them like scalars/vectors, you can see the diff of last commit

Reply:
which commit? I dont see any commits from you or a PR?